rankers/SimpleAggregateRanker/tests/test_aggregate_ranker.py

```diff
@@ -0,0 +1,166 @@
+import numpy as np
+
+from jina.executors.rankers import Chunk2DocRanker
+import pytest
+from .. import SimpleAggregateRanker
+
+
+def chunk_scores(is_positive_score=True):
+    query_chunk2match_chunk = {
+        100: [
+            {'parent_id': 1, 'id': 10, 'score': 0.4, 'length': 200},
+        ],
+        101: [
+            {'parent_id': 2, 'id': 20, 'score': 0.25, 'length': 200},
+            {'parent_id': 2, 'id': 21, 'score': 0.25, 'length': 200},
+            {'parent_id': 2, 'id': 22, 'score': 0.15, 'length': 200},
+            {'parent_id': 2, 'id': 23, 'score': 0.15, 'length': 200},
+        ],
+        110: [
+            {'parent_id': 1, 'id': 10, 'score': 0.5, 'length': 200},
+            {'parent_id': 1, 'id': 11, 'score': 0.2, 'length': 200},
+            {'parent_id': 3, 'id': 20, 'score': 0.1, 'length': 300},
+        ]
+    }
+    query_chunk_meta = {}
+    match_chunk_meta = {}
+    match_idx = []
+    num_query_chunks = len(query_chunk2match_chunk)
+    for query_chunk_id, matches in query_chunk2match_chunk.items():
+        query_chunk_meta[query_chunk_id] = {'length': num_query_chunks}
+        for c in matches:
+            match_chunk_meta[c['id']] = {'length': c['length']}
+            sign = 1 if is_positive_score else -1
+            match_idx.append((
+                c['parent_id'],
+                c['id'],
+                query_chunk_id,
+                c['score'] * sign,
+            ))
+
+    match_idx_numpy = np.array(
+        match_idx,
+        dtype=[
+            (Chunk2DocRanker.COL_MATCH_PARENT_HASH, np.int64),
+            (Chunk2DocRanker.COL_MATCH_HASH, np.int64),
+            (Chunk2DocRanker.COL_DOC_CHUNK_HASH, np.int64),
+            (Chunk2DocRanker.COL_SCORE, np.float64)
+        ]
+    )
+    return match_idx_numpy, query_chunk_meta, match_chunk_meta
+
+
+def assert_document_order(doc_idx):
+    for i in range(0, len(doc_idx) - 1):
+        assert doc_idx[i][1] > doc_idx[i + 1][1]
+
+
+@pytest.mark.parametrize("chunk_scores,aggregate_function,is_reversed_score,doc_ids,doc_scores", [
```

Comment:
single quotes and space between  if possible

Reply:
done